tools/merger.py

```diff
@@ -5,32 +5,35 @@
 
 # todo: can speed up by using info about how many halos instead concatenating
 
-def extract_superslab(fns):
-    z_in = extract_redshift(fns)
-    return np.array([np.int(fn.split('z%.3f.'%z_in)[1].split('.asdf')[0]) for fn in fns])
-    
-def extract_redshift(fns):
-    redshift = float(fns[0].split('z')[-1][:5])
+
+def extract_superslab(fn):
+    # looks like "associations_z0.100.0.asdf"
+    return int(fn.split('.')[-2])
+
+
+def extract_redshift(fn):
+    redshift = float('.'.join(fn.split("z")[-1].split('.')[:2]))
     return redshift
```

Comment:
I think the `[:5]` is a little dangerous on the redshift.  `z10.123` has 6 characters after "z", for example.

Reply:
makes sense! didn't realize we have z > 10 saved

Reply:
We don't for the primary sims!  So it would have worked.  But some of our "extension" sims like HighZ and ScaleFree go higher.  Best to be general!